app/main.py

```diff
@@ -1,8 +1,36 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, list_car: list) -> float:
+        price = 0
+        for car in list_car:
+            if car.clean_mark < self.clean_power:
+                price += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(price, 1)
+
+    def wash_single_car(self, other: object) -> None:
+        other.clean_mark = self.clean_power
+
+    def rate_service(self, grade: int) -> None:
+        self.count_of_ratings += 1
+        self.average_rating = round(((self.count_of_ratings - 1)
```

Comment:
Move the calculation of `average_rating` to a separate variable and then round this variable during `self.average_rating` assignment. It will allow you to avoid code crowding on this line.